test/check/do_block_has_multiple_lines_test.exs

```diff
@@ -0,0 +1,234 @@
+        def validate_coupon(), do: :ok
+
+        def list_by_ids(courier_company_ids),
+          do: where(CourierCompany, [company], company.id in ^courier_company_ids)
+
+        def build_error_message(purchase, _attrs)
+          when purchase.product.is_shippable == false,
+          do: "Purchase's product is not shippable"
+
+        def create_voucher(attrs),
+         do: [1, 2, 3]
+
+         def create_voucher(attrs),
+          do:
+            %Voucher{}
+      end
+      """
+
+      module_source_code
+      |> to_source_file()
+      |> run_check(DoBlockHasMultipleLines)
+      |> refute_issues()
+    end
+  end
+
+  describe "given all the do: functions are INVALID" do
+    test "reports an issue on all instances" do
+      module_source_code = """
+      defmodule CredoSampleModule do
+
+        def create_voucher(attrs),
+          do:
+            %Voucher{}
+            |> change_voucher(attrs)
+
+        def create_voucher(attrs), do: [
+          1
+        ]
+      end
+      """
+
+      module_source_code
+      |> to_source_file()
+      |> run_check(DoBlockHasMultipleLines)
+      |> assert_issues(fn issues -> assert Enum.count(issues) == 2 end)
+    end
+  end
+
+  describe "given all the do: private functions are valid" do
+    test "does NOT report an issue" do
+      module_source_code = """
+      defmodule CredoSampleModule do
+        defp validate_coupon(), do: :ok
+
+        defp list_by_ids(courier_company_ids),
+          do: where(CourierCompany, [company], company.id in ^courier_company_ids)
+
+        defp build_error_message(purchase, _attrs)
+          when purchase.product.is_shippable == false,
+          do: "Purchase's product is not shippable"
+
+        defp create_voucher(attrs),
+         do: [1, 2, 3]
+      end
+      """
+
+      module_source_code
+      |> to_source_file()
+      |> run_check(DoBlockHasMultipleLines)
+      |> refute_issues()
+    end
+  end
+
+  describe "given all the do: private functions are INVALID" do
+    test "reports an issue on all instances" do
+      module_source_code = """
+      defmodule CredoSampleModule do
+
+        defp create_voucher(attrs),
+          do:
+            %Voucher{}
+            |> change_voucher(attrs)
+            |> Repo.insert()
+
+        defp create_voucher(attrs), do: [
+          1,
+          2,
+          3
+        ]
+      end
+      """
+
+      module_source_code
+      |> to_source_file()
+      |> run_check(DoBlockHasMultipleLines)
+      |> assert_issues(fn issues -> assert Enum.count(issues) == 2 end)
+    end
+  end
+
+  describe "given all the if: expressions are valid" do
+    test "does NOT report an issue" do
+      module_source_code = """
+      defmodule CredoSampleModule do
+        if true, do: :ok
+
+        if true,
+          do: :ok
+
+        if true,
+          do:
+            :ok
+
+        if true,
+          do:
+            %Voucher{},
+          else:
+            :ok
+      end
```

Comment:
:roll_eyes: 
```suggestion
        end
```

Reply:
This is the correct indentation for `defmodule`, the keyword list syntax doesn't have end 😛

Reply:
Fixed an indent issue with the module source string on another line here [99c28b9](https://github.com/nimblehq/compass-credo-plugin/pull/27/commits/99c28b93a1f84b4b455304f42597bf6936325413)

👍